api/models/faqDao.js

```diff
@@ -0,0 +1,37 @@
+const { dataSource } = require('./data-source');
+
+const getFaq = async (keyword) => {
+  const stateWhere = `WHERE (question like '%${keyword}%' OR answer like '%${keyword}%')`;
+
+  const result = await dataSource.query(
+    `
+    SELECT
+      id,
+      question,
+      answer
+    FROM 
+      frequently_asked_questions
+    ${stateWhere} 
+    `
+  );
+  return result;
+};
+
+const sendQuestion = async (file_url, title, content, userId) => {
```

Comment:
```suggestion
const createQuestion = async (file_url, title, content, userId) => {
```
Dao에서 만큼은 데이터를 기준으로 네이밍을 해주세요!